APIs/OData.qmd

```diff
@@ -480,6 +480,52 @@ For example, a quicklook for product `S3A_SL_2_FRP____20200821T042815_20200821T0
 
 Download link is also available under *DownloadLink* parameter in Assets.
 
+## Listing product nodes
+
+Product content can be listed by accessing the following URL patterns using Nodes:
+
+```
+https://catalogue.dataspace.copernicus.eu/odata/v1/Products(<PRODUCT_UUID>)/Nodes
+https://catalogue.dataspace.copernicus.eu/odata/v1/Products(<PRODUCT_UUID>)/Nodes(<NODE_NAME>)/Nodes
+https://catalogue.dataspace.copernicus.eu/odata/v1/Products(<PRODUCT_UUID>)/Nodes(<NODE_NAME>)/Nodes(<NODE_NAME>)/Nodes
+```
+
+where:
+
+<PRODUCT_UUID> - is ID of the product obtained by search query,
+
+<NODE_NAME> - is name of element inside product returned from previous listing response.
+
+Only nodes that are folders can have their contents listed. Attempting to list Nodes for a file results returning of empty list. Listing Nodes feature is available for both authorized and unauthorized users
```

Comment:
Only nodes that are folders can have their contents listed. Attempting to list Nodes for file results returning an empty list. The listing Nodes feature is available for both authorized and unauthorized users.